File: nRF.py
import RPi.GPIO as GPIO
import time
 
led_pin = 24
 
GPIO.setwarnings(False)
GPIO.setmode(GPIO.BCM)
GPIO.setup(led_pin,GPIO.OUT)
 
print "linker led pin 24 (BCM GPIO)"
 
while True:
	GPIO.output(led_pin,True)
	print "ON!"
	time.sleep(5)
	GPIO.output(led_pin,False)
	print "OFF!"
	time.sleep(5)

File: nRF.c
#include <wiringPi.h>
#include <stdio.h>
#include <stdlib.h>
 
int led_pin = 5;
 
int main ()
{
if (wiringPiSetup() == -1)
{
printf(¡°Setup wiringPi failed!¡±);
return 1;
}
printf(¡°linker_led pin : GPIO%d (wiringPi pin)\n¡±,led_pin);
pinMode(led_pin, OUTPUT); // set mode to output
 
while(1) 
{
digitalWrite(led_pin, 1); // output a high level 
delay(200);
digitalWrite(led_pin, 0); // output a low level 
delay(200);
}
 
return 0;
}